app/src/main/kotlin/org/cru/godtools/ui/dashboard/lessons/LessonsLayout.kt

```diff
@@ -29,19 +31,27 @@ internal sealed interface DashboardLessonsEvent {
 @Composable
 @OptIn(ExperimentalFoundationApi::class)
 internal fun LessonsLayout(viewModel: LessonsViewModel = viewModel(), onEvent: (DashboardLessonsEvent) -> Unit = {}) {
-    val lessons by viewModel.lessons.collectAsState()
+    val lessons by viewModel.lessons.collectAsState(emptyList())
+    val selectedLanguage by viewModel.selectedLanguage.collectAsState(Language(viewModel.settings.appLanguage))
```

Comment:
now that `viewModel.selectedLanguage` is a `StateFlow` you no longer need to pass an initial value to `collectAsState()`